test/k8sT/DatapathConfiguration.go

```diff
@@ -444,13 +446,23 @@ var _ = Describe("K8sDatapathConfig", func() {
 	})
 })
 
+func testPodConnectivityAcrossNodesMTU(kubectl *helpers.Kubectl) bool {
+	result, _ := testPodConnectivityAndReturnIP(kubectl, true, 1, 1422)
```

Comment:
I suspect that depending on where we run these tests (eg GKE), these MTU numbers may vary because of the base MTU in the environment? Were you trying to take that into account here?

(Also, the numbers seem magic to me; it'd be nice to have constant declaration with a brief summary of how the number was calculated so it can be amended if necessary in future)

Reply:
Its the magic number of bytes in the overhead. We should probably calculate it from mtu on the device correctly though.